.gitignore

```diff
@@ -1,19 +1,20 @@
-#ignore misc BYOND files
-Thumbs.db
-*.log
-*.rsc
-*.int
-*.dmb
-*.lk
-*.backup
-*.before
-data/
-cfg/
-build_log.txt
-
-__pycache__/
-*.py[cod]
-*$py.class
-/.atom-build.json
-.vscode/
-.vs/
+#ignore misc BYOND files
+Thumbs.db
+*.log
+*.rsc
+*.int
+*.dmb
+*.lk
+*.backup
+*.before
+data/
+cfg/
+build_log.txt
+
+__pycache__/
+*.py[cod]
+*$py.class
+/.atom-build.json
+.vscode/
+.vs/
+html/changelogs/example - Copy.yml
```

Comment:
This wasn't here before

Reply:
Thank you for catching it! Should be fixed!